src/hardware/disney.cpp

```diff
@@ -29,6 +29,8 @@
 
 #define DISNEY_SIZE 128
 
+constexpr uint8_t DISNEY_INIT_STATUS = 0b1000'0100; // 0x84
```

Comment:
I suggest using simply 0x84 when binary literal is not good enough…

Placing a comment introduces a risk that someone changes the value without updating the comment.

Reply:
The initial state is coming from the OR'ing of the 0x80 and 0x04 bit states and setting them high.

I've added TODOs for each remaining of a magic number and instead define them by name, for the role they represent. 

For example, maybe 0x80 represents stereo mode, and 0x04 irq high. 

Once these are identified, we would use them instead:

`DS_INITIAL_STATE = STEREO_BIT | IRQ_BIT`

So it's even clearer what state the device is being initialized in. 

> Placing a comment introduces a risk that someone changes the value without updating the comment.

That's a generic risk of all comments in the face of lazy future maintenance.  

If you're wondering why I've retained the magic numbers as comments against the equivalent bit-values: it's to ease maintenance if upstream changes the code, as they are likely to retain the cryptic magic numbers and leave them undefined. So if they move things around, it will be easy to follow the connection using the named bit values.